opm/output/eclipse/InteHEAD.cpp

```diff
@@ -680,7 +680,8 @@ Opm::RestartIO::InteHEAD::tuningParam(const TuningPar& tunpar)
 
 Opm::RestartIO::InteHEAD&
 Opm::RestartIO::InteHEAD::variousParam(const int version,
-                                       const int iprog)
+                                       const int iprog,
+                                       const int num_tracer_comps)
```

Comment:
Since `num_tracer_comps` became unused in the latest update, maybe we could tag it as `[[maybe_unused]]` pending resolution on item 240?

Reply:
Good point! Done ✔️